{"name": "test_login_failure", "status": "passed", "start": 1732760542390, "stop": 1732760542703, "uuid": "c01bf1d3-9ef6-457b-9d44-f321de69127e", "historyId": "875475173a8d8fcaadc0586d31e090c2", "testCaseId": "875475173a8d8fcaadc0586d31e090c2", "fullName": "tests.test_login.TestLogin#test_login_failure", "labels": [{"name": "story", "value": "Неуспешный логин с неверными данными"}, {"name": "feature", "value": "Логин пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "admins-MacBook-Pro.local"}, {"name": "thread", "value": "25675-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}